aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -388,6 +429,8 @@ public AwsProperties(Map<String, String> properties) {
     this.glueLakeFormationEnabled = PropertyUtil.propertyAsBoolean(properties,
         GLUE_LAKEFORMATION_ENABLED,
         GLUE_LAKEFORMATION_ENABLED_DEFAULT);
+    this.tableName = properties.get(LAKE_FORMATION_TABLE_NAME);
```

Comment:
we should not get the table and namespace name here, this should be retrieved when creating `newTableOps`, otherwise it's always going to be null

Reply:
Few doubts:
1. For table level S3 tags, do we need to create new FileIOs for each `newTableOps` call (when LF is not enabled)? The current condition is:
```
protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
    if (catalogProperties != null) {
    ...
    }
```
2. The `tableIdentifier` doesn't have table UUID right now. Should I add the same in the table identifier object?